summarization/stamp_page_picking/budgeted_max_cover_solver.py

```diff
@@ -51,6 +53,10 @@ def find_approximate_maximum_cover(self):
         upper_bound_for_budget = self._sum_of_costs_for_covers(
             list(range(len(self.list_of_covers))))
 
+        # smallest change is 1/(score for the stamp page with maximum score)
+        delta = 1 / StampPageType.get_stamp_type_score(
+                StampPageType.MEDIA_WITH_TEXT_AND_TITLE)
+
```

Comment:
While previously defined increment for lower_bound/upper_bound was not wrong this value is more accurate.
The "delta" value should be the smallest increment possible. In this case it is `1 / (Score for maximum stamp)`